doc/classes/Vector2.xml

```diff
@@ -339,25 +341,25 @@
 			Enumerated value for the Y axis.
 		</constant>
 		<constant name="ZERO" value="Vector2( 0, 0 )">
-			Zero vector.
+			Zero vector, a vector with all components set to [code]0[/code].
 		</constant>
 		<constant name="ONE" value="Vector2( 1, 1 )">
-			One vector.
+			One vector, a vector with all components set to [code]1[/code].
 		</constant>
 		<constant name="INF" value="Vector2( inf, inf )">
-			Infinity vector.
+			Infinity vector, a vector with all components set to [code]INF[/code].
```

Comment:
Could link to `[constant @GDScript.INF]` (but that's not portable for other langs).